### PR TITLE
feat: Switch to manifest v3

### PR DESCRIPTION
**Pull request by fl0rek**
waiting for lumina-node/lumina-node-wasm release

### CODE REVIEW

File: src/popup.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
 // example of NodeClient showing off starting node and monitoring its syncing status
 
-import init, { NodeClient, NodeConfig, Network } from "lumina-node-wasm";
+import { NodeClient, NodeConfig, Network } from "lumina-node-wasm";
 
 // must be called before using any of the functionality imported from wasm
-await init();
+//await init();
```
Comment (zvolin) [inline src/popup.js]: remove it

File: package.json [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
   },
   "devDependencies": {
     "copy-webpack-plugin": "^12.0.2",
+    "prettier": "3.3.3",
     "webpack-cli": "^5.1.4"
   },
   "dependencies": {
-    "lumina-node": "^0.3.0"
+    "lumina-node": "^0.3.0",
+    "lumina-node-wasm": "file:../lumina-main/node-wasm/pkg/lumina-node-wasm-0.2.0.tgz"
```
Comment (zvolin) [inline package.json]:
```suggestion
    "lumina-node": "^0.5.0",
    "lumina-node-wasm": "^0.5.0"
```

File: package.json [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
+    "fmt": "prettier -w ."
   },
   "devDependencies": {
     "copy-webpack-plugin": "^12.0.2",
+    "prettier": "3.3.3",
     "webpack-cli": "^5.1.4"
   },
   "dependencies": {
-    "lumina-node": "^0.3.0"
+    "lumina-node": "^0.5.0",
```
Comment (zvolin) [inline package.json:12]: actually we no longer use lumina-node

**Review by zvolin — APPROVED** [review]
:+1:

**Review by oblique — APPROVED** [review]

**Pull request merged**